### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/response-directly.md`

### PR DESCRIPTION
**Pull request by Joao-Pedro-P-Holanda**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/advanced/response-directly.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Retornando uma Resposta Diretamente
+
+Quando você cria uma *operação de rota* no **FastAPI** você pode retornar qualquer dado nela: um dicionário (`dict`), uma lista (`list`), um modelo do Pydantic ou banco de dados, etc.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/response-directly.md:3]: It got a little weird. It looks like you can return the database

"*um modelo do Pydantic ou banco de dados*"

I would add a "*ou do do banco*"

File: docs/pt/docs/advanced/response-directly.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Mas você pode retornar a `JSONResponse` diretamente nas suas *operações de rota*.
+
+Pode ser útil para retornar cabeçalhos e cookies personalizados, por exemplo.
+
+## Retornando uma `Response`
+
+Na verdade, você pode retornar qualquer `Response` ou subclasse dela.
+
+/// tip | Dica
```
Comment (ceb10n) [inline docs/pt/docs/advanced/response-directly.md:17]: I really don't know if the call-outs need or don't need double quotes. But all the translations that I see uses the double quotes for the translated part, as you can see in:

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/environment-variables.md?plain=1#L3

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/virtual-environments.md?plain=1#L5

https://github.com/fastapi/fastapi/blob/10f3cb5ab1d5e2848146b85256df056c1bcc9753/docs/pt/docs/python-types.md?plain=1#L15

and etc...

File: docs/pt/docs/advanced/response-directly.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+## Notas
+
+Quando você retorna uma `Response` diretamente os dados não são validados, convertidos (serializados) ou documentados automaticamente.
+
+Mas você ainda pode documentar como descrito em [Retornos Adicionais no OpenAPI
+](additional-responses.md){.internal-link target=_blank}.
+
+Você pode ver nas próximas seção como usar/declarar essas `Responses` customizadas enquanto mantém a conversão e documentação automática dos dados.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/response-directly.md:70]: missing the plural of "seção"

"*nas próximas seções*"

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great! 🚀

File: docs/pt/docs/advanced/response-directly.md (hunk 2 above)
Comment (ceb10n) [reply docs/pt/docs/advanced/response-directly.md:17]: I searched here, and apparently the docs used to use the version with !!! Tip "Dica". Back then, it used the double quotes. But now we don't need it anymore.

Just ignore this reviewed line

Comment (Joao-Pedro-P-Holanda) [reply docs/pt/docs/advanced/response-directly.md:17]: I also found it weird when i saw the double quotes being rendered in the page, maybe it's an update on mkdocs.

**Review by ceb10n — APPROVED** [review]
✨

**Review by cassiobotaro — APPROVED** [review]

**Comment by alejsdev** [comment]
Thank you! :sunglasses: @Joao-Pedro-P-Holanda @ceb10n @cassiobotaro @nazarepiedady

**Pull request merged**